src/coupling/assembly_base.hh

```diff
@@ -46,7 +46,7 @@ public:
     }
 
     /// Assembles the volume integrals on cell.
-    inline void cell_integral(FMT_UNUSED DHCellAccessor cell) {}
+    inline void cell_integral(FMT_UNUSED unsigned int element_patch_idx, FMT_UNUSED unsigned int dh_local_idx) {}
```

Comment:
It makes cell_integral method (call of point range) faster. We can't use same optimization in other types of integrals because they need access to equivalent points at lower dim element throught neighbour object etc.

Reply:
Points range call is fine. Just try to measure passing the cell or cell reference vs. new parameters.

Reply:
OK, new parameters are not faster then passing the cell. Changes partly revert.